python/test/unit/language/test_core_amd.py

```diff
@@ -2202,13 +2202,14 @@ def __str__(self):
 
 
 class MfmaLayout:
-    def __init__(self, non_k_dim, warps_per_cta, isTransposed):
+    def __init__(self, non_k_dim, k_dim, warps_per_cta, isTransposed):
```

Comment:
So why does mfmaLayout has kDim? I think `kDim` belongs to dotOperandEncoding, right?

Reply:
good catch!
tests passed and I didn't notice this is unused